tests/phpunit/api/v4/Action/CreateCustomValueTest.php

```diff
@@ -73,4 +75,75 @@ public function testGetWithCustomData() {
     $this->assertEquals($optionValues, $createdOptionValues);
   }
 
+  /**
+   * Test setting/getting a multivalue customfield with date only
+   */
+  public function testCustomDataWithDateOnly() {
+    CustomGroup::create(FALSE)
+      ->addValue('title', 'MyContactDateFields')
+      ->addValue('name', 'MyContactDateFields')
+      ->addValue('extends', 'Contact')
+      ->addValue('is_multiple', TRUE)
+      ->execute()
+      ->first();
+
+    CustomField::create(FALSE)
+      ->addValue('custom_group_id:name', 'MyContactDateFields')
+      ->addValue('label', 'Date field')
+      ->addValue('name', 'date_field')
+      ->addValue('data_type:name', 'Date')
+      ->addValue('html_type:name', 'Select Date')
+      ->addValue('date_format:name', 'yy-mm-dd (2009-12-31)')
```

Comment:
This is a separate issue, but I looked at this code and suddenly realized that APIv4 thinks translatable labels are the same thing as machine names. Yikes! That's definitely wrong and these 3 lines should not work. I've put up #22730 which will fix the bug (and break this api call, it should be updated to not use the `:name` suffix anyway).